web/src/redux/slices/ResultList.slice.ts

```diff
@@ -0,0 +1,57 @@
+import {createAsyncThunk, createSlice, PayloadAction} from '@reduxjs/toolkit';
+import TestRunResultGateway from '../../gateways/TestRunResult.gateway';
+import TraceService from '../../services/Trace.service';
+import {IAssertionResult, TAssertionResultList} from '../../types/Assertion.types';
+import {ITest} from '../../types/Test.types';
+import {ITrace} from '../../types/Trace.types';
+
+interface ITestResultListState {
+  resultListMap: Record<string, IAssertionResult[]>;
+}
```

Comment:
New slice that stores the result list info, in the future this will come from the BE